src/physics/em/BremsstrahlungProcess.hh

```diff
@@ -27,13 +37,15 @@ class BremsstrahlungProcess : public Process
     using SPConstParticles = std::shared_ptr<const ParticleParams>;
     using SPConstMaterials = std::shared_ptr<const MaterialParams>;
     using SPConstImported  = std::shared_ptr<const ImportedProcesses>;
+    using Options          = const BremsstrahlungOptions;
```

Comment:
Rather than a typedef (which for clarity shouldn't have the included `const` anyway) we should make that struct a nested `Options` like we do with PhysicsParams. The main reason *not* to do that would be if this class were templated, which it isn't.

Reply:
Thanks! Change BremsstrahlungOptions to BremsstrahlungProcess::Options.